usersimcrs/items/ratings.py

```diff
@@ -112,3 +115,23 @@ def get_random_user_id(self) -> str:
             User ID.
         """
         return random.choice(list(self._user_ratings.keys()))
+
+    def create_split(
+        self, historical_ratio: float
+    ) -> Tuple["Ratings", "Ratings"]:
+        """Splits ratings into historical and ground truth ratings.
```

Comment:
Can you explain the difference between historical and ground thruth ratings?

Reply:
We base the simulator in actual user behavior (ratings): preferences are established based on "historical" ratings (80% of the data). The ground truth part is held-out data that could be used to estimate whether a user would like a recommendation or not. Simulation usually goes until the user receives a recommendation that they'd like; that'd mean an item with >=4 stars in the ground_truth portion. Perhaps ground_truth is not the right term here. Also, this second part is missing from the interaction model.